kodilla-rps/src/main/java/com/kodilla/rps/RpsGame.java

```diff
@@ -0,0 +1,41 @@
+package com.kodilla.rps;
+
+public class RpsGame {
+    private static RpsGameDefinition rpsGameDefinition;
+    private static int roundNumber;
+
+    public RpsGame(RpsGameDefinition rpsGameDefinition) {
+        this.rpsGameDefinition = rpsGameDefinition;
+        this.roundNumber = 1;
+    }
+
+    public RpsGameDefinition getRpsGameDefinition() {
+        return this.rpsGameDefinition;
+    }
+
+    public static void play() {
+        while (roundNumber <= rpsGameDefinition.getRounds()) {
+
+            RpsMenu.printRoundInfo(roundNumber);
+            RpsMenu.printShortMenu();
+            RpsTurn userTurn = RpsMenu.getUserTurn();
+
+            if (userTurn == RpsTurn.ROCK || userTurn == RpsTurn.PAPER || userTurn == RpsTurn.SCISSORS ) {
+                RpsRoundRunner.run(userTurn);
```

Comment:
tu dobrze by bylo stworzyc obiekt rundy new RpsRound(roundNumber) i odpalic metode run. to w rundzie odbywac sie ma pojedyncza gra.